src/fdc/MSXtar.hh

```diff
@@ -66,6 +66,9 @@ private:
 	void chroot(std::string_view newRootDir, bool createDir);
 
 private:
+	unsigned getStartCluster(const MSXDirEntry& entry) const;
```

Comment:
Consider annotating this function with `[[nodiscard]]`.
I didn't check, maybe other functions could be annotated as well?